src/app/tile/tile-back/tile-back.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'aar-tile-back',
+  templateUrl: './tile-back.component.html',
+  styleUrls: ['./tile-back.component.css']
+})
+export class TileBackComponent implements OnInit {
+
+  @Input() imageName: string;
+  url: string;
+  constructor() { }
+
+  ngOnInit() {
+    this.url = "\\assets\\tile-images\\" + this.imageName;
```

Comment:
Skulle kunna använda [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):

```suggestion
    this.url = `\\assets\\tile-images\\${this.imageName}`;
```